setup.py

```diff
@@ -1,6 +1,44 @@
-from distutils.core import setup
+# -*- coding: utf-8 -*-
 
-setup(name='viziphant',
-      version='0.0.1',
-      packages=['viziphant']
-      )
+import os
+
+from setuptools import setup
+
+with open(os.path.join(os.path.dirname(__file__),
+                       "viziphant", "VERSION")) as version_file:
+    version = version_file.read().strip()
+
+with open("README.md") as f:
+    long_description = f.read()
+with open('requirements/requirements.txt') as fp:
+    install_requires = fp.read()
+extras_require = {}
+for extra in ['docs', 'tests']:
+    with open('requirements/requirements-{0}.txt'.format(extra)) as fp:
+        extras_require[extra] = fp.read()
+
+
+setup(
+    name="viziphant",
+    version=version,
+    packages=['viziphant', 'viziphant.tests'],
+    include_package_data=True,
+    install_requires=install_requires,
+    extras_require=extras_require,
+    author="Viziphant authors and contributors",
+    author_email="contact@python-elephant.org",
+    description="Viziphant is a package for the visualization of the analysis"
+                "results of electrophysiology data in Python",
+    long_description=long_description,
+    license="BSD",
+    url='https://github.com/INM-6/viziphant',
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: BSD License',
+        'Natural Language :: English',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2',
```

Comment:
we should remove this line and state somewhere that viziphant works with python 3.6+ only

Reply:
Addressed with 25d70c9ec1d09f24d1d6e54b5764c8985f99f671

Reply:
I have also added python>=3.6 to the requirements.txt in 3eef25a99806a18760d48af5357318ed86b3e06d